benchto-driver/src/test/resources/application.yaml

```diff
@@ -1,5 +1,5 @@
 benchmarks: src/test/resources/benchmarks
-sql: src/test/resources/sql
+sql: src/test/resources/sql,src/test/resources/sql-second-dir
```

Comment:
i really really _do_ hope this can be set on the command line. Otherwise I have hard time imagining how we use together with our internal benchmarks holding project.